src/features/restaurants/restaurantsSlice.tsx

```diff
@@ -0,0 +1,44 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit'
+import { CleanedRestaurantsState } from '../../app/store/interfaces'
+import { fetchRestaurants } from '../../utilities/APICalls'
+import { cleanList } from '../../utilities/utilities'
+
+export interface ListState {
+  restaurants: CleanedRestaurantsState[]
+  status: string
+  error: string | null
+}
+
+const initialState = {
+  restaurants: [] as CleanedRestaurantsState[],
```

Comment:
This was the piece we needed for TypeScript to stop yelling at us on the payload, right? Awesome job figuring this out!

Reply:
Yup! Because we had it only as an empty, it ignored our interface above and `defaulted` it to a `never[]` type. This and our hours not being formatted properly were causing issues. Also, because our state for restaurants was empty, I kept getting type errors with trying to push, so I changed it to setting the restaurants from `getRestaurants` and not pushing into the array directly (on line 35)